scraper/src/strategies/default_strategy.py

```diff
@@ -187,6 +189,21 @@ def get_records_from_dom(self, current_page_url=None):
                 record['no_variables'] = record['url'] == record[
                     'url_without_variables']
 
+            # Define our own ObjectID to enable proper analytics
+            hierarchy_to_hash = {}
+            for lvl in hierarchy:
+                if hierarchy[lvl] is not None:
+                    hierarchy_to_hash[lvl] = hierarchy[lvl]
+            raw_hash = hashlib.sha1(json.dumps(
+                {'hierarchy_to_hash': json.dumps(hierarchy_to_hash,
```

Comment:
Remove this second call `json.dumps`, it is unnecessary:
```python
json.dumps(
  {'hierarchy_to_hash': hierarchy_to_hash,
   'url': record['url'],
   'position': position}, sort_keys=True).encode('utf-8')
```

Reply:
Good point. Thanks